tools/clang/test/HLSLFileCheck/samples/SimpleHs12.hlsl

```diff
@@ -62,7 +61,7 @@ HSPerPatchData HSPerPatchFunc( const InputPatch< PSSceneIn, 12 > points, OutputP
 [partitioning("fractional_odd")]
 [outputtopology("triangle_cw")]
 [patchconstantfunc("HSPerPatchFunc")]
-[outputcontrolpoints(3)]
```

Comment:
Assuming this had to be changed just because it's an earlier return now?

Reply:
Yes that's correct. One is Sema, the other is at a later stage in the DXC side.

Reply:
I don't follow that explanation. This is an end-to-end compilation. Sema is still being invoked, why isn't the error firing anymore?

Reply:
Sorry, mixed up things:
the error is indeed firing as expected. But the check being moved to Sema, I added a new test (`tools/clang/test/CodeGenDXIL/semantic.hull-output.size-mismatch.hs.hlsl`) which covers it from `-verify`, hence didn't wanted to check it again here using `FileCheck`

Reply:
Makes sense, reverted this.

Reply:
So, turns out I confused my confusion:
The error doesn't fire anymore if I don't fix the `outputcontrolpoint` first.
The `outputcontrolpoint` is a SEMA error, while the other is a DXIL codegen error.
This means the compilation stops earlier, so we don't show both errors at the same time.